presentation/src/main/java/com/anytypeio/anytype/presentation/sets/ViewerEditWidgetUi.kt

```diff
@@ -0,0 +1,95 @@
+package com.anytypeio.anytype.presentation.sets
+
+import com.anytypeio.anytype.core_models.DVViewer
+import com.anytypeio.anytype.core_models.DVViewerType
+import com.anytypeio.anytype.core_models.Id
+import com.anytypeio.anytype.core_models.ObjectTypeIds
+import com.anytypeio.anytype.core_models.ObjectWrapper
+import com.anytypeio.anytype.domain.objects.StoreOfObjectTypes
+import com.anytypeio.anytype.domain.objects.StoreOfRelations
+
+data class ViewerEditWidgetUi(
+    val showWidget: Boolean,
+    val id: Id? = null,
+    val name: String = "",
+    val viewerId: Id,
+    val defaultObjectType: ObjectWrapper.Type?,
+    val isDefaultObjectTypeEnabled: Boolean = false,
+    val layout: DVViewerType?,
+    val relations: List<String> = emptyList(),
```

Comment:
please use typealias - not clear what is string here

Reply:
done